CAN-D/Src/usbd_desc.c

```diff
@@ -5,95 +5,100 @@
-};
-
-#if defined ( __ICCARM__ ) /* IAR Compiler */
-  #pragma data_alignment=4
+    {
+        USBD_FS_DeviceDescriptor, USBD_FS_LangIDStrDescriptor, USBD_FS_ManufacturerStrDescriptor, USBD_FS_ProductStrDescriptor, USBD_FS_SerialStrDescriptor, USBD_FS_ConfigStrDescriptor, USBD_FS_InterfaceStrDescriptor};
+
+#if defined(__ICCARM__) /* IAR Compiler */
+#pragma data_alignment = 4
 #endif /* defined ( __ICCARM__ ) */
 /** USB standard device descriptor. */
 __ALIGN_BEGIN uint8_t USBD_FS_DeviceDesc[USB_LEN_DEV_DESC] __ALIGN_END =
-{
-  0x12,                       /*bLength */
-  USB_DESC_TYPE_DEVICE,       /*bDescriptorType*/
-  0x00,                       /*bcdUSB */
-  0x02,
-  0x02,                       /*bDeviceClass*/
-  0x02,                       /*bDeviceSubClass*/
-  0x00,                       /*bDeviceProtocol*/
-  USB_MAX_EP0_SIZE,           /*bMaxPacketSize*/
-  LOBYTE(USBD_VID),           /*idVendor*/
-  HIBYTE(USBD_VID),           /*idVendor*/
-  LOBYTE(USBD_PID_FS),        /*idProduct*/
-  HIBYTE(USBD_PID_FS),        /*idProduct*/
-  0x00,                       /*bcdDevice rel. 2.00*/
-  0x02,
-  USBD_IDX_MFC_STR,           /*Index of manufacturer  string*/
-  USBD_IDX_PRODUCT_STR,       /*Index of product string*/
-  USBD_IDX_SERIAL_STR,        /*Index of serial number string*/
-  USBD_MAX_NUM_CONFIGURATION  /*bNumConfigurations*/
-}; /* USB_DeviceDescriptor */
-
-#if defined ( __ICCARM__ ) /* IAR Compiler */
-  #pragma data_alignment=4
+    {
+        0x12,                 /*bLength */
+        USB_DESC_TYPE_DEVICE, /*bDescriptorType*/
+        0x00,                 /*bcdUSB */
+        0x02,
+        0x02,                /*bDeviceClass*/
+        0x02,                /*bDeviceSubClass*/
+        0x00,                /*bDeviceProtocol*/
+        USB_MAX_EP0_SIZE,    /*bMaxPacketSize*/
+        LOBYTE(USBD_VID),    /*idVendor*/
+        HIBYTE(USBD_VID),    /*idVendor*/
+        LOBYTE(USBD_PID_FS), /*idProduct*/
+        HIBYTE(USBD_PID_FS), /*idProduct*/
+        0x00,                /*bcdDevice rel. 2.00*/
+        0x02,
+        USBD_IDX_MFC_STR,          /*Index of manufacturer  string*/
+        USBD_IDX_PRODUCT_STR,      /*Index of product string*/
+        USBD_IDX_SERIAL_STR,       /*Index of serial number string*/
+        USBD_MAX_NUM_CONFIGURATION /*bNumConfigurations*/
+};                                 /* USB_DeviceDescriptor */
+
+#if defined(__ICCARM__) /* IAR Compiler */
+#pragma data_alignment = 4
 #endif /* defined ( __ICCARM__ ) */
 
 /** USB lang indentifier descriptor. */
 __ALIGN_BEGIN uint8_t USBD_LangIDDesc[USB_LEN_LANGID_STR_DESC] __ALIGN_END =
-{
-     USB_LEN_LANGID_STR_DESC,
-     USB_DESC_TYPE_STRING,
-     LOBYTE(USBD_LANGID_STRING),
-     HIBYTE(USBD_LANGID_STRING)
-};
-
-#if defined ( __ICCARM__ ) /* IAR Compiler */
-  #pragma data_alignment=4
+    {
+        USB_LEN_LANGID_STR_DESC,
+        USB_DESC_TYPE_STRING,
+        LOBYTE(USBD_LANGID_STRING),
+        HIBYTE(USBD_LANGID_STRING)};
```

Comment:
This is strange, I don't know if I like how this ended up being formatted -- specifically the `}`